strongbox-distribution/src/main/resources/deb/postinst

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e
```

Comment:
This would not cover any `pipe fails` (which could happen at lines `5` and `6`). 
To cover those cases you'd need to do `set -euo pipefail`. :)

Reply:
sh is the shell that is supposed to be used for this, and pipefail is not available in sh. 

I will get back to this.

Reply:
Oh, you're right - my bad. I don't remember if `/bin/bash` was available on all Debian/Ubuntu based distributions by default so maybe we should leave it as it is then. If you happen to find any way to make it quit at the first error it gets that would be good but otherwise - we can just merge it as it is for now.

Just FYI - [this](https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/) is how you'd do it in `/bin/bash`.